app/src/main/java/com/android/periodpals/model/alert/Alert.kt

```diff
@@ -0,0 +1,36 @@
+package com.android.periodpals.model.alert
+
+import java.time.LocalDateTime
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Alert(
+    @SerialName("idAlert") val idAlert: String,
+    @SerialName("idProfile") val idProfile: String,
+    @SerialName("name") val name: String,
+    @SerialName("product") val product: Product,
+    @SerialName("urgency") val urgency: Urgency,
+    @SerialName("createdAt") @Contextual val createdAt: LocalDateTime,
+    @SerialName("location") val location: String, // TODO: Create data class Location
+    @SerialName("message") val message: String,
+    @SerialName("status") val status: Status
+)
+
+enum class Product {
+  TAMPON,
+  PAD
+}
+
+enum class Urgency {
+  URGENT,
+  NORMAL,
+  NOT_URGENT,
+}
```

Comment:
Good idea to have them in an `enum` class. However, I don't think these are the correct labels. Looking at the Figma, it seems that it should be high, medium and low. 

I also suggest inverting the order since `enum` classes essentially map a name to an `int`, starting from zero. What you have here would map `URGENT` to 0, `NORMAL` to 1 and `NOT_URGENT` to 2, but it would be more intuitive to have it the other way around.

```suggestion
enum class Urgency {
  LOW,
  MEDIUM,
  HIGH,
}
```